src/wallet.cpp

```diff
@@ -113,6 +113,16 @@ CPubKey CWallet::GenerateNewKey()
     return pubkey;
 }
 
+CBitcoinAddress CWallet::GenerateNewAutoMintKey()
+{
+    CBitcoinAddress btcAddress;
+    CKeyID keyID = GenerateNewKey().GetID();
+    btcAddress.Set(keyID);
+    CWalletDB(strWalletFile).WriteAutoConvertKey(btcAddress);
+    SetAddressBook(keyID, "automint-address", "receive");
+    return btcAddress;
```

Comment:
Don't you need to add the address to `setAutoConvertAddresses` in there?

Reply:
good catch, see 2159a58